great_expectations/checkpoint/types/checkpoint_result.py

```diff
@@ -11,6 +10,9 @@
 from great_expectations.data_context.types.base import (
     CheckpointConfig,
     CheckpointConfigSchema,
+    delete_runtime_parameters_batch_data_references_from_config,
```

Comment:
could/should these not be methods on the config object?

Reply:
@donaldheppner In fact, even better: in the next commit, they will be moved to `batch.py` core module.

Reply:
Thank you @alexsherstinsky, if that is the case, could I ask that individual tests for the methods be added in the next commit (or is it PR?)